### ISSUE TITLE
/learn Section

### ISSUE THREAD

**Issue by thequbit** [mediapublic/mediapublic#5]
_Note: this is the contents of a number of comments from another repo.  User's names have been place above the comments to keep a history of what was said._

---

*_@melodykramer *_

From gitter.im: "I was thinking earlier about process, and how it's so important to document and share process - particularly for public sector work. There's no one place to do that. There's no place for people to say "I used this tool for this project" or "We experimented with email newsletters and this happened" or "We used this open source repo and were able to do this." I wonder if the Learn section could become that."

---

**@thequbit**

I've been picturing the /learn section as kind of a 'pinned blog posts' area.  But more like 'pinned essay' area.  Is that what you were thinking @melodykramer ?

---

**@melodykramer**

Yup. Based on user research / interviews:

**Here's why this section is important**

There is no one place in public media (or the public sector) for people to share information, code bases, and tools. There are currently dozens of Facebook groups, Slack channels and listservs where people currently post information. These are siloed, walled, and undiscoverable - and terrible to surface archival content. 

**tldr** This section should serve as the place where people share stories, tools, and open source software projects and explain how they worked.

All users should be able to:
1. Read blog posts
2. Sort blog posts by subject
3. Search archives
4. Share blog posts

Signed in users should be able to:
1. Create, edit, and submit blog posts 
2. Assign keywords to blog posts
3. Comment / annotate blog posts

Super signed in users should be able to:
1. Approve blog posts
2. Edit blog posts

---

**@thequbit**

Re-formatting this to match the auth levels outlined in #11 and the end-points defined in #9:

**VIEWER**
- Read Howtos
- Sort Howtos by subject
- Search Howtos
- Share Howtos to social media

**STANDARD_USER**
- All things **VIEWER** can do
- Create, Edit, and Submit a Howto
- Assign keywords to any Howto ( others and their own )
  - Keywords need to be approved??
- Comment on Howtos
- Annotate Howtos
  - So doing something like comments associated with paragraphs and be able to turn those on and off would be amazing.  I think I have a picture in my mind of what this could look like.  I think Disqus may already support it

**ELEVATED_USER**
- All things **STANDARD_USER** can do
- Approve Howtos to be published

Note that I changed _learn_ to _howto_.  This is just how I wrote the database structure.  100% okay with changing it to whatever vocab term we want to use.  I don't think we should use `blog` though, because I think we're going to want to have a legit blog that we are posting weekly or so to drive traffic to the site ( Content Marketing and all that ).

That seem good?

---

*_@melodykramer *_

yes, agreed.

**Comment by GabeIsman** [comment]
I think we've got a good handle on this now.
